Localization/Resources/es-ES/winget.resw

```diff
@@ -1268,4 +1275,78 @@ Especifique uno de ellos con la opción '--source' para continuar.</value>
   <data name="SystemArchitecture" xml:space="preserve">
     <value>Arquitectura del sistema</value>
   </data>
+  <data name="PreserveArgumentDescription" xml:space="preserve">
+    <value>Conserva todos los archivos y directorios creados por el paquete (archivo portátil)</value>
+  </data>
+  <data name="PurgeArgumentDescription" xml:space="preserve">
+    <value>Elimina todos los archivos y directorios del directorio del paquete (archivo portátil)</value>
+  </data>
+  <data name="RenameArgumentDescription" xml:space="preserve">
+    <value>Valor para cambiar el nombre del archivo ejecutable (archivo portátil)</value>
+  </data>
+  <data name="WaitArgumentDescription" xml:space="preserve">
+    <value>Pide al usuario que presione cualquier tecla antes de salir</value>
+  </data>
+  <data name="InstallerElevationExpected" xml:space="preserve">
+    <value>El instalador solicitará que se ejecute como administrador; se espera un mensaje.</value>
+  </data>
+  <data name="InstallerProhibitsElevation" xml:space="preserve">
+    <value>El instalador no se puede ejecutar desde un contexto de administrador.</value>
+  </data>
+  <data name="ModifiedPathRequiresShellRestart" xml:space="preserve">
+    <value>Variable de entorno PATH modificada; reinicie el shell para usar el nuevo valor.</value>
+  </data>
+  <data name="ProductCodeArgumentDescription" xml:space="preserve">
+    <value>Filtros con el código de producto</value>
```

Comment:
The original message was "filters using the product code". I think "filters" here is a verb and not a noun, right? As in "it filters" and not "these are filters". If so, this should be "Filtra".

Reply:
Yes, you are correct, filters in this scenario is used as a verb.